bpf/lib/common.h

```diff
@@ -531,6 +532,7 @@ enum {
 	SVC_FLAG_AFFINITY     = (1 << 4),  /* sessionAffinity=clientIP */
 	SVC_FLAG_LOADBALANCER = (1 << 5),  /* LoadBalancer service */
 	SVC_FLAG_ROUTABLE     = (1 << 6),  /* Not a surrogate/ClusterIP entry */
+	SVC_FLAG_CHECK_SRC    = (1 << 7),  /* Check LoadBalancer source range */
```

Comment:
nit: It'd be good to make these a common transformation of the userspace name of the flag so it's easy to cross-reference (eg `serviceFlagSourceRange` -> `SVC_FLAG_SOURCE_RANGE`).